src/app/components/playlist/playlist.rs

```diff
@@ -16,6 +16,16 @@ pub trait PlaylistModel {
 
     fn current_song_id(&self) -> Option<String>;
 
+    // NOTE: `is_playing` is for if there is *any* song being played
+    //       `playlist_is_playing` is for if the specific playlist is being played
+    fn is_playing(&self) -> bool {
```

Comment:
like I said earlier, no need to add these to the traits! just add the relevant impl blocks to details.rs and playlist_details.rs

Reply:
To clarify, is it alright if I put it on the DetailsModel impl in `details_model.rs`? I think that makes sense?